src/main/java/org/apache/bcel/classfile/StackMapType.java

```diff
@@ -29,9 +29,9 @@
  * @see StackMap
  * @see Const
  */
-public final class StackMapType implements Cloneable {
+public final class StackMapType implements Node, Cloneable {
 
-    public static final StackMapType[] EMPTY_ARRAY = {}; // must be public because BCELifier code generator writes calls to it
+    static final StackMapType[] EMPTY_ARRAY = {}; // package visibility as BCELifier code generator writes calls to constructor translating null to EMPTY_ARRAY
```

Comment:
You can't make something public, not public, this will break BC, you can't tell now because we are in the middle of a release.

Reply:
Ah sorry about this. Putting it back to public.